sdk/servicebus/service-bus/package.json

```diff
@@ -75,9 +75,10 @@
     ]
   },
   "dependencies": {
-    "@azure/core-amqp": "1.0.0-preview.3",
```

Comment:
Change to use latest of `core-amqp` is being done as part of #6861
Any reason to include this in this PR?

Reply:
No reason, ideally that should have been merged already - branch will get updated once that's done

Reply:
Feel free to use the `allowedAlternativeVersions` section in the `common-versions.json` file to get rush to allow you to use different versions of `@azure/core-amqp` in Service Bus vs Event Hubs.

This PR is for the Azure identity work and it would help if the changes here are restricted to using the `@azure/identity` library in the constructors.

If #6861 gets merged before this PR, we can then
- rebase your branch from `feature/service-bus-track-2`
- revert the change to `common-versions.json` file

Reply:
I looked over the diffs - the duplicate changes related to core-amqp are very few and can be ignored - it's this one line change updating version, and the type casting ones in files `messageReceiver.ts`, `messageSender.ts` and `batchingReceiver.ts`. (You can ignore the files altogether if they already appear in #6861)

All other 24-25 files are relevant and can be reviewed. The build succeeds as well.